spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -387,6 +387,13 @@ func TestSample(t *testing.T) {
 	assertContains(t, out, "Updated data to VenueDetails column\n")
 	out = runSample(t, queryWithJsonParameter, dbName, "failed to query with json parameter")
 	assertContains(t, out, "The venue details for venue id 19")
+
+	out = runSample(t, createTableWithForeignKeyDeleteCascade, dbName, "failed to create table with foreign key delete constraint")
```

Comment:
issue: create a new test file or method for this sample.

See:
https://googlecloudplatform.github.io/samples-style-guide/#dedicated-testing-per-sample

Reply:
Done